src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
+
+export async function delComment(id : number) {
+  const result = await fetch(`${BASE_URL}/comments/${id}`, { method: 'DELETE' });
+
+  return result;
+}
+
+export async function postComment(
+  name : string,
+  email : string,
+  body : string,
+  postId: number,
```

Comment:
You are duplicating the Comment interface. I would suggest using `<Omit>` here:
```suggestion
export async function postComment({ name, email, body, postId}: Omit<Comment, "id">
```